README.md

```diff
@@ -15,24 +15,32 @@ Contributors include:
 
 To get started with this project:
 
-1. Launch SBT:
+1. Get the [Bower](https://bower.io/) dependencies, to go folder `src/main/resources/webapp`
```

Comment:
If you don't want to install Bower then we can skip this step, by adding all the 2k+ files in the repo once. But it won't be the correct way to do it.